docs/cookbook/basic-data-import.md

```diff
@@ -193,7 +193,8 @@ class ProductRepository extends BaseProductRepository
 }
 ```
 
-Add information about the class extension into the container configuration in [`services.yml`](../../project-base/src/Shopsys/ShopBundle/Resources/config/services.yml)
+Add information about the class extension into the container configuration in [`services.yml`](../../project-base/src/Shopsys/ShopBundle/Resources/config/services.yml).
+You need to add the same line into [`services_test.yml`](../../project-base/src/Shopsys/ShopBundle/Resources/config/services_test.yml) as well to [make the service public in TEST environment](../introduction/faq-and-common-issues.md#what-is-the-configuration-file-services_testyml-good-for).
```

Comment:
I suggest to invert this sentence - "To make the service public in test environment, you need to add..." because it is optional change.

And the same think in next 2 occurences